include/xcfun.h

```diff
@@ -40,7 +40,7 @@ enum xc_mode
       XC_N_S,
       // GGA with gradient squares
       XC_A_GAA,
```

Comment:
Since you're at it, you could document all `enum`-s using the Doxygen style comments for the betterment of posterity.